salt/journal-cms/config/srv-journal-config-local.settings.php

```diff
@@ -72,6 +72,13 @@
 $settings['file_private_path'] = '{{ pillar.journal_cms.files.private_path }}';
 {% endif %}
 
+{% if pillar.journal_cms.journal.base_uri %}
+$settings['journal_path'] = '{{ pillar.journal_cms.journal.base_uri }}';
```

Comment:
The build uses the default pillars from https://github.com/elifesciences/builder-private/blob/master/pillar/journal-cms.sls so some fake values need to be added there (and for all other environments that will actually use it: `end2end`, `continuumtest` and `prod`).

Reply:
I will submit a PR to `builder-private`.

Reply:
Current failure is what we expect :+1: 
```
[54.237.238.159] out:           ID: site-settings
[54.237.238.159] out:     Function: file.managed
[54.237.238.159] out:         Name: /srv/journal-cms/config/local.settings.php
[54.237.238.159] out:       Result: False
[54.237.238.159] out:      Comment: Unable to manage file: Jinja variable 'salt.utils.odict.OrderedDict object' has no attribute 'journal'
[54.237.238.159] out:      Started: 08:50:17.098966
[54.237.238.159] out:     Duration: 36.299 ms
[54.237.238.159] out:      Changes:  
```